cmd/openshift-install/create.go

```diff
@@ -268,13 +268,16 @@ func destroyBootstrap(ctx context.Context, config *rest.Config, directory string
 			}
 
 			logrus.Debugf("added %s: %s", event.Name, event.Message)
-			return event.Name == "bootstrap-complete", nil
+			return event.Name == "bootstrap-success", nil
 		},
 	)
 	if err != nil {
-		return errors.Wrap(err, "waiting for bootstrap-complete")
+		return errors.Wrap(err, "waiting for bootstrap-success")
 	}
 
+	logrus.Info("Waiting 30 seconds for the production control-plane to enter the load balancer")
```

Comment:
Here you have to send tear down event. Otherwise no draining will happen.

Reply:
> Here you have to send tear down event. Otherwise no draining will happen.

This isn't for draining.  This is waiting for the production control plane to enter the load balancer.  [The 10-second sleep in the destroy logic][1] is for draining.

[1]: https://github.com/openshift/installer/pull/1148#discussion_r252070613